src/sagemaker/transformer.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, model_name, instance_count, instance_type, strategy=None, ass
         self.sagemaker_session = sagemaker_session or Session()
 
     def transform(self, data, data_type='S3Prefix', content_type=None, compression_type=None, split_type=None,
-                  job_name=None):
+                  job_name=None, wait=True, logs=True):
```

Comment:
I think this might constitute a breaking change since this method used to not be blocking, but I do like mimicking the behavior of fit()...

@mvsusp for a second opinion?

Reply:
I do like mimicking the behavior of fit as well. IMO it is okay to add `wait` and `logs` with default values if these defaults keep the original behaviour, i.e. `wait=False`.

Reply:
We need do add an integration test with `wait=true` as well.